evap/staff/forms.py

```diff
@@ -917,20 +917,26 @@ class QuestionnairesAssignForm(forms.Form):
     def __init__(self, *args, course_types, **kwargs):
         super().__init__(*args, **kwargs)
 
-        for course_type in course_types:
-            self.fields[course_type.name] = forms.ModelMultipleChoiceField(
-                required=False,
-                queryset=Questionnaire.objects.general_questionnaires().exclude(
-                    visibility=Questionnaire.Visibility.HIDDEN
-                ),
-            )
         contributor_questionnaires = Questionnaire.objects.contributor_questionnaires().exclude(
             visibility=Questionnaire.Visibility.HIDDEN
         )
-        self.fields["all-contributors"] = forms.ModelMultipleChoiceField(
-            label=_("All contributors"), required=False, queryset=contributor_questionnaires
+        general_questionnaires = Questionnaire.objects.general_questionnaires().exclude(
+            visibility=Questionnaire.Visibility.HIDDEN
         )
 
+        for course_type in course_types:
+            self.fields["general-" + course_type.name] = forms.ModelMultipleChoiceField(
```

Comment:
`course_type.name` is the translated human-readable version of the name and can easily contain spaces. Using `-` as a delimiter here while appending that name is weird. Using a translated name is weird. Maybe just use the id? (Repeated below)